thunderfish/consistentfishes.py

```diff
@@ -2,23 +2,26 @@
 
 def extract_fundamentals(fishlists):
```

Comment:
Please move this function to the harmonicgroups module. It really belongs there.